setup.py

```diff
@@ -11,5 +11,6 @@
       description='Mongorm',
       long_description='Mongorm',
       platforms=['any'],
-      install_requires=['pymongo >= 2.4.2', 'pysignals'],
+      install_requires=['pymongo >= 2.4.2', 'pysignals', 'iso8601'],
```

Comment:
I wonder if actually pymongo 2.9+ is required (since we updated to support mongo 3.0)

Reply:
There's a few comments about using pymongo 3.0+ specific things already it seems (as in breaking changes that mean a version < 3.0 would break). I just think everyones been blissfully unaware because this `setup.py` pretty much just installs the latest version of pymongo (`3.9.0` at the current time). So yeah, I think I'll update this to be a hard `>= 3.0.0` requirement